src/api/app/controllers/webui/request_controller.rb

```diff
@@ -345,9 +345,12 @@ def changes_diff
     if @action.target_package_object&.file_exists?(filename, { rev: target_rev, expand: 1 }.compact)
       target_file = project_package_file_path(@action.target_project_object, @action.target_package_object, filename, rev: target_rev, expand: 1)
     end
+    diff = sourcediff.dig('files', filename, 'diff', '_content')
+    disabled = diff.nil? && sourcediff.dig('files', filename, 'state').in?(%w[deleted added])
+    diff = "#{sourcediff.dig('files', filename, 'diff', 'lines')} lines skipped" if diff.nil? && !disabled
```

Comment:
I think we decided to not display the `nnn lines skipped`. In such cases we can keep it the accordion disabled 

Meaning you can completely ignore this line

`diff = "#{sourcediff.dig('files', filename, 'diff', 'lines')} lines skipped" if diff.nil? && !disabled`

Reply:
Removed